cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -174,6 +174,7 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         try {
             repository.delete(releaseName_build)
         } catch (_: Exception) {
+            /* Exception intentionally ignored: folder might not exist */
```

The catch block originally was empty, which violates SonarQube rules by not explaining why the exception is ignored. The expert instructions required a comment to be added, thereby clarifying that ignoring the exception is intentional because the folder might not exist. This change explicitly documents the intent, making the code easier to maintain and satisfy static analysis rules.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)